packages/appeals-service-api/src/services/user.service.js

```diff
@@ -61,10 +62,11 @@ const createUser = async (user) => {
 		throw ApiError.badRequest();
 	}
 
-	user.enabled = true;
+	user.enabled = true; //todo: decide what to do with this field in light of status field
 	user.createdAt = new Date();
 
 	if (!user.isAdmin) {
+		user.status = 'added';
```

Comment:
This would be better as a string constant somewhere that we use across files so that if we change the value it will change everywhere, it also makes the code and tests less brittle if there is ever a typo in  a string for example.
Instead something like user.status = userStatus.added

Javascript doesn't properly support enum I don't think, but in most other languages this would be an enum.